### PR TITLE
Example Project: Next.JS & Tailwind CSS

### PR DESCRIPTION
**Pull request by spreston4**
This PR adds an example project that incorporates Shipwright Tokens into a project built with Next.JS & Tailwind CSS.

To run the project, cd into `example/project-nextjs-tailwind` then `npm run dev`. The project will be viewable at `http://localhost:3000/`.

This example hardcodes values / repeats code that in order to show users who may not be familiar with Tailwind CSS how to utilize their custom classes in their own project.

### CODE REVIEW

File: example/project-nextjs-tailwind/components/colors.js [hunk 1]
```diff
@@ -0,0 +1,243 @@
+const Colors = () => {
+  return (
+    <div className="flex flex-col items-center w-3/4 pb-3 mb-5 border-b-2">
+      <p className="mb-3 h4 text-type-black-secondary">Colors</p>
+      <div className="flex flex-col items-center mb-3">
+        <p className="h5">Primary</p>
```
Comment (panzacoder) [inline example/project-nextjs-tailwind/components/colors.js:6]: Just for semantics, can you update the tags that use a heading css component to be the right html tag?

in this case: 
```suggestion
        <h5 className="h5">Primary</h5>
```

File: example/project-nextjs-tailwind/components/header.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          objectFit="cover"
+          src={Logo}
+          alt="Make Waves"
+          layout="fill"
+          priority
+        />
+      </div>
+      <h1 className="h4 text-type-black-secondary">
+        Shipwright Tokens - Next JS / Tailwind CSS
+      </h1>
```
Comment (panzacoder) [inline example/project-nextjs-tailwind/components/header.js:19]: Same in these files, styles should match tags
```suggestion
      <h4 className="h4 text-type-black-secondary">
        Shipwright Tokens - Next JS / Tailwind CSS
      </h4>
```

**Review by panzacoder — CHANGES_REQUESTED** [review]
Couldn't leave this comment in-line, but I think out .github, README.md, and tokens folders should all be within the project-nextjs-tailwind directory

Looking good otherwise!

File: example/project-nextjs-tailwind/components/header.js (hunk 2 above)
Comment (spreston4) [reply example/project-nextjs-tailwind/components/header.js:19]: Yup good call. Also the directory issue is weird - must have made a mistake when copying the files over

**Review by panzacoder — APPROVED** [review]
Looks good!

**Pull request merged**